### DISCUSSION TITLE
［灌水］今天继续分享好康的

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#5194] [灌水]
![Screenshot_2024-09-19-11-24-44-658_com miui home](https://github.com/user-attachments/assets/af3b41b4-60be-4ac5-aa6c-3557a0a3249b)
这是一个漫画平台😄懂得都懂扫码下载
![mmexport1726716331254](https://github.com/user-attachments/assets/e3bb870d-4f2a-4d85-86f6-488a4e1438a3)

**Comment by Qilai7** [comment]
6

**Comment by BunnYbota** [comment]
我就知道

**Comment by 2ningmeng** [comment]
...举报了

**Reply by fzdqz** [reply]
上钩了😉

**Reply by Yizakl** [reply]
传奇钓鱼佬@fzdqz

**Reply by KELERSAMA** [reply]
传奇大佬 @Yizakl <img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_9213146029750742963.png" alt="阿姬倒地了02" width="75" >

**Reply by XinRenShenJiangZiShu** [reply]
经过我的追本溯源，这个是假的，其真正内容为社会主义核心价值观的视频，来自于被大多数人称之为B站的软件

**Reply by emmGrinch** [reply]
不信 @XinRenShenJiangZiShu

**Comment by Yizakl** [comment]
@KELERSAMA 我是fw喵～

**Comment by lllliufeng** [comment]
感谢分享，很好康，再接再厉请

**Comment by fzdqz** [comment]
你们说的都对，但是哔哩哔哩就不能是个漫画平台吗？∠( ᐛ 」∠)_唉嘿

**Comment by cunish-k** [comment]
这个粉红软件和我的不太一样

**Comment by Yizakl** [comment]
@fzdqz 
# 你 又 钓 上 鱼 了！！！

**Reply by fzdqz** [reply]
在哪◝₍ᴑ̑ДO͝₎◞

**Reply by Yizakl** [reply]
![IMG_0128](https://github.com/user-attachments/assets/0e05c231-52c3-4af4-8f91-2ff3ddb6450e)

**Comment by hjkxiyi** [comment]
你应该把哔咔的图片删除，然后说是一个粉色的可以看漫画的软件
